README.md

```diff
@@ -15,75 +15,78 @@
+
+Why and how Hops is structured into modules can be found [here](./docs/modules.md) and [here is a more detailed and complete treatise on its concepts](./docs/concepts.md).
 
-Otherwise:
+### Quick start
 
 ```shell
-npm install --global hops
-hops init my-awesome-project
+npx hops init my-awesome-project
 cd my-awesome-project
 npm start
 ```
 
 This will start hops in development mode. Visit [http://localhost:8080](http://localhost:8080) to see your app in the browser and make some changes to the code in your editor to see it live-reloading.
 
-If you want to build the JS bundles for production, run `npm run build` in your terminal and to start a Node.js server just type in `npm start --production`.
+You can find a complete walk through in our [Getting Started](./docs/getting-started.md).
 
-### Project Templates
+### FAQ
 
-Hops comes with a couple of different project templates you can use with `hops init --template`. If you do not explicitly select one, `hops init` defaults to `hops-template-react`.
+#### When is Hops the framework for you?
 
-| Package                                                                                     | NPM                                                                                                                   |
-| ------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| [Hops Template Minimal](https://github.com/xing/hops/tree/master/packages/template-minimal) | [![npm](https://img.shields.io/npm/v/hops-template-minimal.svg)](https://www.npmjs.com/package/hops-template-minimal) |
-| [Hops Template React](https://github.com/xing/hops/tree/master/packages/template-react)     | [![npm](https://img.shields.io/npm/v/hops-template-react.svg)](https://www.npmjs.com/package/hops-template-react)     |
-| [Hops Template GraphQL](https://github.com/xing/hops/tree/master/packages/template-graphql) | [![npm](https://img.shields.io/npm/v/hops-template-graphql.svg)](https://www.npmjs.com/package/hops-template-graphql) |
+When you want to share a consistent setting over more than one project. Either now or in the future. This applies to companies with more than one development team or when you constantly set up projects that should be consistent, e.g. training projects, tutorials, consultant work, etc.
 
-If, for example, you want to bootstrap your project to use both React and GraphQL, you will have to call `hops init` like this:
+Even without striving for consistency across multiple projects you may want to use Hops for a (single) new project when you do not want to write/copy/maintain your own webpack configuration. We feel your pain and have done that for you. However, we give you all the customization you may beed using our configuration package.
```

Comment:
Typo: beed -> need
Plus, I'm not completely sure about that sentence. Would it make more sense to say:
`However, we give you all the customization you may need by providing you a configuration package.`
?

Reply:
How about: "...we try hard to give you all the customizability you may ever need."